scalameta/semanticdb/shared/src/main/scala/scala/meta/internal/semanticdb/package.scala

```diff
@@ -18,8 +18,8 @@ package object semanticdb {
   }
 
   object schema {
-    val Document = org.langmeta.internal.semanticdb.schema.Document
```

Comment:
Why are we maintaining forwarders for internal APIs?

Reply:
I guess, for perfect compatibility with whatever was in the `scala.meta` package at the point of writing. If you'd like we can kill this.